### PR TITLE
Fix parsing create notification track owner id

### PR DESCRIPTION
**Pull request by sabrina-kiam**
### Description
Create notifications for tracks weren't showing up in discovery UI 🤦 not sure how we missed this. a while ago joe changed the specifier of track create notifications which audiusbackend was expecting for tracks, so had to update audiusbackend to parse for the information elsewhere in the notif

### CODE REVIEW

**Comment by sabrina-kiam** [comment]
Screenshot 2023-04-26 at 9 20 00 PM
Screenshot 2023-04-26 at 9 19 25 PM

File: packages/common/src/services/audius-backend/AudiusBackend.ts [hunk 1]
```diff
@@ -2339,7 +2339,14 @@ export const audiusBackend = ({
         })
         .flat()
         .filter(removeNullable)
-      const userId = decodeHashId(notification.actions[0].specifier) as number
+      // playlist owner ids are the specifier of the playlist create notif
+      const userId =
+        entityType !== Entity.Track
```
Comment (raymondjacobson) [inline packages/common/src/services/audius-backend/AudiusBackend.ts:2344]: can entity type be something besides track & collection? will this break if so?

**Review by raymondjacobson — APPROVED** [review]

File: packages/common/src/services/audius-backend/AudiusBackend.ts (hunk 1 above)
Comment (isaacsolo) [reply packages/common/src/services/audius-backend/AudiusBackend.ts:2344]: 'create' types only happen on track and playlist handler

maybe use === and reverse the logic for clarity

Comment (sabrina-kiam) [reply packages/common/src/services/audius-backend/AudiusBackend.ts:2344]: also `entityType` var is created on line 2327 and only set in 3 places, once on initialization, and in lines 2332 and 2337 where its all either set to track, playlist, or album

Comment (sabrina-kiam) [reply packages/common/src/services/audius-backend/AudiusBackend.ts:2344]: the code could definitely be more safeguarded

**Review by dylanjeffers — APPROVED** [review]
Nice catch

**Pull request merged**